dvc/repo/__init__.py

```diff
@@ -129,19 +129,24 @@ def __repr__(self):
 
     @classmethod
     def find_root(cls, root=None):
-        if root is None:
-            root = os.getcwd()
-        else:
-            root = os.path.abspath(os.path.realpath(root))
+        root_dir = os.path.abspath(os.path.realpath(root or os.getcwd()))
+
+        if not os.path.isdir(root_dir):
+            raise NotDvcRepoError("directory {} does not exist".format(root))
```

Comment:
We try to enclose generated parts, so it is easier to read logs
```suggestion
            raise NotDvcRepoError("directory '{}' does not exist".format(root))
```

Reply:
Updated the message.